### ISSUE TITLE
sfn list ERROR: NoMethodError

### ISSUE THREAD

**Issue by runningman84** [sparkleformation/sfn#46]
this was a fresh install on a ubuntu 14.04 box with chefdk 0.8
```
user@VirtualBox:~/chefrepo/cloud/sparkle$ sfn print
ERROR: RuntimeError: Template processing is disabled. Path to serialized template via `--file` required!
user@VirtualBox:~/chefrepo/cloud/sparkle$ sfn list
ERROR: NoMethodError: undefined method `keys' for nil:NilClass
```

**Comment by chrisroberts** [comment]
Template processing has been disabled by default mainly as a historical thing. Probably time to flip that default. 

For the `NoMethodError`, I'm guessing it is a lack of configuration. I have a `config` command stubbed out that I'll make some time to actually implement. Will also run through various misconfigured setups to catch cryptic errors like these to make them more user friendly.

**Comment by chrisroberts** [comment]
Changes pushed into dev to provide nice error around this + added flag to `conf` command for generating a basic `.sfn` file. Will close once released. Thanks for raising this issue!

**Comment by chrisroberts** [comment]
Fixed and released as of 1.1.10. Thanks!
